react/features/toolbox/functions.web.js

```diff
@@ -4,6 +4,32 @@ import { hasAvailableDevices } from '../base/devices';
 
 declare var interfaceConfig: Object;
 
+/**
+ * Returns a set of button names to be displayed in the toolbox, based on the screen width.
+ *
+ * @param {number} width - The width of the screen.
+ * @returns {Set} The button set.
+ */
+export function getToolbarAdditionalButtons(width: number): Set<string> {
+    if (width <= 500) {
+        if (width <= 390) {
+            if (width <= 332) {
```

Comment:
Could we extract these values to some meaningfully named constants for a bit more clarity?

Reply:
Done.